Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.GPU.sb.in

```diff
@@ -714,7 +714,11 @@
 )
 
 (when (defined? 'syscall-unix)
+#if !PLATFORM(IOS) || __IPHONE_OS_VERSION_MIN_REQUIRED >= 160000
```

Comment:
This isn't the first case in this file, but I wonder: Why do we test PLATFORM(IOS) in the iOS sandbox file? Isn't it always true?